### PR TITLE
temporarily put a notification on the top of certain scripts

### PR DESCRIPTION
**Pull request by Bjvanminnen**
image

We want to temporarily put this notification on the top of a couple of scripts. This is a short-term pragmatic hack.

### CODE REVIEW

File: apps/src/sites/studio/pages/scriptOverview.js [hunk 1]
```diff
@@ -23,6 +25,30 @@ function initPage() {
...
+  ];
+
+  if (announcementCourses.includes(scriptData.name)) {
+    ReactDOM.render(
+      <Notification
+        type={NotificationType.information}
+        notice="This course has recently been updated!"
+        details="See what changed and how it may affect your classroom"
+        buttonText={i18n.learnMore()}
+        buttonLink="TODO"
```
Comment (Bjvanminnen) [inline apps/src/sites/studio/pages/scriptOverview.js]: Still awaiting this URL

File: apps/src/sites/studio/pages/scriptOverview.js [hunk 2]
```diff
@@ -23,6 +25,30 @@ function initPage() {
     store.dispatch(setVerifiedResources(true));
   }
 
+  // This notification is a temporary hack we want to stick on a few courses. We
+  // expect to remove it in the near future, and build a longer term solution
+  // for adding notifications to specific scripts that is LB based.
+  const announcementCourses = [
```
Comment (islemaster) [inline apps/src/sites/studio/pages/scriptOverview.js:31]: image

Just kidding.  :+1: :shipit:

File: apps/src/sites/studio/pages/scriptOverview.js [hunk 3]
```diff
@@ -23,6 +25,30 @@ function initPage() {
...
+    'coursef',
+    'express'
+  ];
+
+  if (announcementCourses.includes(scriptData.name)) {
+    ReactDOM.render(
+      <Notification
+        type={NotificationType.information}
+        notice="This course has recently been updated!"
+        details="See what changed and how it may affect your classroom"
```
Comment (islemaster) [inline apps/src/sites/studio/pages/scriptOverview.js]: We're cool with not localizing this text?

Comment (islemaster) [reply apps/src/sites/studio/pages/scriptOverview.js]: Nit: I would put a period at the end of this sentence.

**Pull request merged**